kubectl-mayactl/cli/util/types.go

```diff
@@ -23,14 +23,38 @@ type Volume struct {
 	Version                 string
 }
 
+//VolumeInfo struct will have all the details we want to give in the output for
+// mayactl command volume describe
+type VolumeInfo struct {
+	AccessMode              []corev1.PersistentVolumeAccessMode
+	Capacity                string
+	CSPC                    string
+	CSIDriver               string
+	CSIVolumeAttachmentName string
+	Name                    string
+	Namespace               string
+	PVC                     string
+	ReplicaCount            int
+	VolumePhase             corev1.PersistentVolumePhase
+	StorageClass            string
+	Version                 string
+	Size                    string
+	Status                  v1.CStorVolumePhase
+}
+
 // PortalInfo keep info about the ISCSI Target Portal.
 type PortalInfo struct {
-	IQN           string
-	VolumeName    string
-	Portal        string
-	Size          string
-	Status        []v1.CStorVolumeCondition
-	ReplicaCount  int
-	ReplicaStatus []v1.ReplicaStatus
-	//ControllerNode string
+	IQN            string
+	VolumeName     string
+	Portal         string
+	TargetIP       string
+	TargetNodeName string
+}
+
+// CStorReplicaInfo holds information about the cstor replicas
+type CStorReplicaInfo struct {
+	Name     string
+	NodeName string
+	ID       v1.ReplicaID
+	Status   string
```

Comment:
We can put comments for each field -- like what it signifies/or what info it gives.